clients/admin-ui/src/features/common/custom-fields/CustomFieldsList.tsx

```diff
@@ -65,6 +63,24 @@ export const CustomFieldsList = ({
                   return null;
                 }
 
+                const name = `customFieldValues.${customFieldDefinition.id}`;
+                if (
+                  !customFieldDefinition.allow_list_id &&
```

Comment:
There isn't an ENUM value for each input type yet. In order to check if a field is open text the field type and allow list id must be checked